index.js

```diff
@@ -0,0 +1,96 @@
+// Popup menu bar
+
+const body = document.querySelector('body');
+const navigation = document.querySelector('.nav-bar');
+const menu = document.querySelector('.logo');
+const menuSec = document.querySelector('.menu-bar');
+const closeButton = document.querySelector('#close-btn');
+const openButton = document.querySelector('#hamburg-open');
+const main = document.querySelector('main');
+
+if (window.innerWidth < 768) {
+  const menuBar = () => {
+    menu.style.display = 'none';
+    navigation.style.boxShadow = 'none';
+    body.style.backgroundColor = '#F07F7E';
+    body.style.overflow = 'hidden';
+    menuSec.style.display = 'block';
+    menuSec.style.position = 'absolute';
+    menuSec.style.top = '200px';
+    menuSec.style.left = '50%';
+    menuSec.style.transform = 'translate(-50%, -50%)';
+    main.style.visibility = 'hidden';
+  };
+  openButton.addEventListener('click', menuBar);
+
+  const closeMenu = () => {
+    body.style.overflow = '';
+    menu.style.display = '';
+    navigation.style.boxShadow = '';
+    body.style.backgroundColor = '';
+    menuSec.style.display = '';
+    main.style.visibility = '';
+  };
+
+  closeButton.addEventListener('click', closeMenu);
+}
```

Comment:
Nice Job so far👍. Currently, your navbar functionalities doesn't work as required. When the hamburger is clicked, a mobile menu should be displayed with the different navigation links😉. currently, when it is clicked, nothing happens. kindly do well to fix this👍.
<img width="780" alt="image" src="https://github.com/2004-okari/CAPSTONE-01/assets/64643995/71ad013d-dc6a-4fe5-8251-7400673d6abd">

https://github.com/2004-okari/CAPSTONE-01/assets/64643995/31c55fa1-e691-4627-8977-7193409d8b0c